Wikipedia/Code/ReadingListDetailViewController.swift

```diff
@@ -566,9 +190,7 @@ extension ReadingListDetailViewController: ReadingListDetailUnderBarViewControll
         title = name
     }
     
-    func readingListDetailUnderBarViewController(_ underBarViewController: ReadingListDetailUnderBarViewController, didBeginEditing textField: UITextField) {
-        editController.isTextEditing = true
```

Comment:
If you set `editController.isTextEditing` to `true` here, the top bar buttons will change to "Cancel" and "Done" when you begin editing a list's title, allowing user to decide if they want to keep the new title or discard changes -

![img_b36b6f4f6a29-1](https://user-images.githubusercontent.com/9299317/50768026-4a683280-127f-11e9-992f-c60ca70395ce.jpeg)